include/ttmlir/Dialect/TTNN/IR/TTNNOpsAttrs.td

```diff
@@ -67,4 +67,14 @@ def TTNN_MemoryConfigAttr : TTNN_Attr<"MemoryConfig", "memory_config"> {
   let assemblyFormat = "`<` params `>`";
 }
 
+def TTNN_MeshShapeAttr : TTNN_Attr<"MeshShape", "mesh_shape"> {
+  let summary = "TTNN Mesh Shape";
+  let description = [{
+    TTNN mesh shape
+  }];
+
+  let parameters = (ins "int64_t":$y, "int64_t":$x);
+  let assemblyFormat = "`[` $y `,` $x `]`";
```

Comment:
Should we use `<` instead of `[` in assembly format to align with other types?

Reply:
@nsmithtt what do you think?